styles/duckduckgo/catppuccin.user.css

```diff
@@ -16,12 +16,15 @@
 ==/UserStyle== */
 
 @-moz-document domain(duckduckgo.com) {
-  :root:not(.dark-bg) {
-    #catppuccin(@lightFlavor, @accentColor);
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
   }
-
-  :root.dark-bg {
-    #catppuccin(@darkFlavor, @accentColor);
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
   }
```

Comment:
Why are you removing the styles for considering the Duckduckgo theme preference? Looks like it still works.

Reply:
i thought it was broken because it kept switching to latte even though i was on dark mode

Reply:
Have you set the Duckduckgo theme toggle to dark?